docker-compose-citus.yml

```diff
@@ -133,6 +203,20 @@ services:
         env_file: docker/patroni.env
         hostname: work2-2
         container_name: demo-work2-2
+        deploy:
+            resources:
+                <<: *resource-limit-psql
+        volumes:
+            - work2_2_data:/var/lib/postgresql/data
         environment:
             <<: *work2_env
             PATRONI_NAME: work2-2
+
+volumes:
+  coord_1_data:
+  coord_2_data:
+  coord_3_data:
+  work1_1_data:
+  work1_2_data:
+  work2_1_data:
+  work2_2_data:
```

Comment:
This compose file exists just to play with Citus/Patroni, not for production.
Therefore we don't want to allocate some unnecessary resources.